crates/cairo-lint-core/tests/test_files/breaks/breaks

```diff
@@ -2,32 +2,32 @@
 
 //! > cairo_code
 fn main() {
-    let mut a = 1_u32;
-    loop {
-        if a == 10 {
-            break ();
-        }
-        a += 1;
-    }
+  let mut a = 1_u32;
```

Comment:
ah yes didn't think about these lints :'( replace the `loop` by `for elt in array![].span()` (it might add another diagnostic but it won't hurt the fix